### PR TITLE
remove failure

### PR DESCRIPTION
**Pull request by fredszaq**

### CODE REVIEW

File: ffi-convert/Cargo.toml [hunk 1]
```diff
@@ -10,6 +10,6 @@ readme = "../README.md"
 keywords = ["ffi"]
 
 [dependencies]
-ffi-convert-derive = "0.2.2"
-failure = "0.1"
+ffi-convert-derive = {path = "../ffi-convert-derive"}
```
Comment (anthonyray) [inline ffi-convert/Cargo.toml:13]: Won't this cause problems when we perform a release for `ffi-convert` ? Should this be overridden in a cargo configuration file ?

**Review by anthonyray — CHANGES_REQUESTED** [review]
Good P.R, feels good to have typed errors ! Just a minor comment in the dependencies of `ffi-convert`.

**Comment by anthonyray** [comment]
This fixes #22

File: ffi-convert/Cargo.toml (hunk 1 above)
Comment (fredszaq) [reply ffi-convert/Cargo.toml:13]: we'll change it back when releasing :P

**Pull request merged**